includes/functions.php

```diff
@@ -1098,6 +1117,8 @@ function tsml_get_meetings($arguments = [], $from_cache = true)
 				'time' => isset($meeting_meta[$post->ID]['time']) ? $meeting_meta[$post->ID]['time'] : null,
 				'end_time' => isset($meeting_meta[$post->ID]['end_time']) ? $meeting_meta[$post->ID]['end_time'] : null,
 				'time_formatted' => isset($meeting_meta[$post->ID]['time']) ? tsml_format_time($meeting_meta[$post->ID]['time']) : null,
+				'feedback_url' => tsml_feedback_url($post),
+				'edit_url' => get_edit_post_link($post),
```

Comment:
this looks good, except could we get it unescaped? i'm seeing `&nbsp;` instead of `&` in the URL: `"https://wordpress.test/wp-admin/post.php?post=213&amp;action=edit"`